### DISCUSSION TITLE
Using CLI instead of bindings

### DISCUSSION THREAD

**Discussion by omertuc** [bootc-dev/bootc#780] [General]
Is there any particular reason why https://github.com/containers/bootc/blob/d17e471eaa1da722bed005dfa820eac76aaf876e/lib/src/install.rs#L603-L606 is using the `os-init` CLI instead of the `ostree_sysroot_init_osname` libostree API?

**Comment by omertuc** [comment]
@cgwalters ?

**Reply by cgwalters** [reply]
It honestly may be that some of this logic I was going back and forth with the old https://github.com/coreos/coreos-assembler/blob/38235218542c39a29255933d3cea64e91754e85f/src/create_disk.sh and may have just started out forking as it was a simpler translation.

Happy to switch to the API!

**Reply by omertuc** [reply]
Alright, made #781
